docs/en/faq.md

```diff
@@ -17,6 +17,17 @@ If the data pipeline meets the below criteria, Redshift serverless is preferred.
 - Redshift is not used for streaming ingestion.
 - The estimated cost is lower than the provisioned cluster.
 
+### I already enable data modeling on Redshift, so why can't I see the schema and tables created by this solution in the Redshift query editor?
+
+This solution creates a separate database and schema within your Amazon Redshift cluster for storing and processing clickstream events. By default, the schema, tables, and views are only owned by the user who created them and are not visible to users who log into the Redshift [query editor][redshift-query-editor].
+
```

Comment:
and are not visible to users who log into the Redshift --> and are not visible to **_other_** users who log into the Redshift Query Editor.